### DISCUSSION TITLE
Is there a way to use the same ContentView for each Tab?

### DISCUSSION THREAD

**Discussion by needledrag** [roubachof/Sharpnado.Tabs#67] [Q&A]
Hi all,

Just wondering if there's a way to have a number of tabs but use the same view  in the ViewSwitcher for each TabItem (basically they're all require the same layout but the viewable data changes based on the selected tab index).

Any suggestions would be appreciated, thanks
Shaun

**Comment by Cruik** [comment]
Have exact the same requirement. Anyone an idea?

**Comment by roubachof** [comment]
You don't have to use a view switcher, just use a regular view container and do things based on the selected tab.
